src/controllers/user.controller.js

```diff
@@ -0,0 +1,90 @@
+const userService = require('./../services/user.service');
+
+const getUsers = async (req, res) => {
+  try {
+    const users = await userService.getAllUsers();
+
+    res.send(users);
+  } catch (error) {
+    res.status(500).send('Server error');
```

Comment:
Consider adding more detailed error messages or logging the error details in the catch block to help with debugging and monitoring.